src/python/grpcio_channelz/setup.py

```diff
@@ -47,10 +47,7 @@ def run(self):
 CLASSIFIERS = [
     'Development Status :: 5 - Production/Stable',
     'Programming Language :: Python',
-    'Programming Language :: Python :: 2',
-    'Programming Language :: Python :: 2.7',
     'Programming Language :: Python :: 3',
-    'Programming Language :: Python :: 3.4',
     'Programming Language :: Python :: 3.5',
     'Programming Language :: Python :: 3.6',
```

Comment:
Should we add `3.7` for grpcio-* packages?

Reply:
Oversight on my part. Done.